zerver/migrations/0516_alter_groupgroupmembership_subgroup_and_more.py

```diff
@@ -0,0 +1,31 @@
+# Generated by Django 5.0.5 on 2024-04-20 15:03
+
+import django.db.models.deletion
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("zerver", "0515_alter_scheduledmessagenotificationemail_mentioned_user_group"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="groupgroupmembership",
+            name="subgroup",
+            field=models.ForeignKey(
+                on_delete=django.db.models.deletion.CASCADE,
+                related_name="+",
+                to="zerver.namedusergroup",
+            ),
+        ),
+        migrations.AlterField(
+            model_name="usergroup",
+            name="direct_subgroups",
+            field=models.ManyToManyField(
+                related_name="direct_supergroups",
+                through="zerver.GroupGroupMembership",
+                to="zerver.namedusergroup",
+            ),
+        ),
+    ]
```

Comment:
Am I correct that these migrations don't generate any SQL? Just worth confirming.

Reply:
Yes, the second `AlterField` call will not generate any SQL.